types/xmldom-reborn/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for xmldom-reborn 1.0
+// Project: https://github.com/MrDatastorage/xmldom-reborn
+// Definitions by: Qubo <https://github.com/tkqubo>
+//                 Andreas Neeven <https://github.com/aneeven>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const DOMParser: DOMParserStatic;
+declare const XMLSerializer: XMLSerializerStatic;
+declare const DOMImplementation: DOMImplementationStatic;
+
+interface DOMImplementationStatic {
+    new(): DOMImplementation;
+}
+
+interface DOMParser {
```

Comment:
From the example usage here it seems like this needs to be class instead of interface.
https://github.com/MrDatastorage/xmldom-reborn#example

Reply:
What I meant is why not declare this as class intead of mixing  interface, class and statics as another interface